umh.docs.umh.app/content/en/docs/production-guide/administration/customize-umh-installation.md

```diff
@@ -35,6 +35,50 @@ should really know what you are doing, as you might break the installation.
 To get more information about the values you can change, see the
 [Helm chart documentation](https://learn.umh.app/docs/core/helmchart/).
 
+## Customize the configuration before the installation {#customize-values}
```

Comment:
that is not true. it works on whatever kube context is set. for example, if the user imported multiple kubeconfig from  remote clusters, `kubectl` will set the context to the latest cluster added, and `helm` will use the same context of `kubectl`. i could add a note about checking what the current context is, to prevent changing the configuration of the wrong cluster

Reply:
I mean that you cannot apply this if you are following the installation tutorials. You cannot customize the configuration when using the flatcar installation (only we can do it). You are only able to do it in the k3d installation, so it would be great if you could align the story. 

For example:

Customize the configuration before k3d installation

OR

create an article explaining how to install it on a random kubernetes cluster as a Helm chart. then you can publish this article, but still should mention that this is not possible with flatcar and that people should use the management console in the future for that.

Reply:
Technically you can still use step 4 of this section even for a flatcar installation. Point is, you are always able to use a custom value file, either before or after install, regarding of the installation method. 

I believe that is important to tell that one can have a config file ready for a specific machine, and in the case that they have multiple identical setup they can reuse the same file without manually editing the values from lens each time. They can just import the kubeconfig on their machine and `helm upgrade` everything with the same file

Let me think about how to say that without leading to confusion about the possibility to configure the values before the installation when using flatcar.

I like the idea of a tutorial about installing umh on an existing kubernetes cluster, it might have some use cases. I'll open a feature request for that.

_Side note_: would be cool to have the option to customize the `values.yaml` file on mgmt console before installation, maybe with a nice frontend with checkboxes for which services to enable and stuff like that. Maybe let's discuss over slack about this